IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -216,8 +216,7 @@ - (void)executeRequestImpl:(MSIDRequestCompletionBlock)completionBlock
             }
         }
     }
-    
-    [self fetchCachedRefreshTokenWithCompletionHandler:^(MSIDBaseToken<MSIDRefreshableToken> *refreshToken, MSIDRefreshTokenTypes tokenType, NSError *error) {
+    [self fetchCachedRefreshTokenInOrderOf:@[@(MSIDAppRefreshTokenType), @(MSIDFamilyRefreshTokenType)] completionHandler:^(MSIDBaseToken<MSIDRefreshableToken> *refreshToken, MSIDRefreshTokenTypes tokenType, NSError *error) {
```

Comment:
since there're only two types, you might simplify this with a boolean.

e.g. fetchCachedTokenAndCheckForFRTFirst:(BOOL)checkForFRT